packages/lesswrong/components/ea-forum/StickiedPosts.tsx

```diff
@@ -7,6 +7,9 @@ const styles = (theme: ThemeType): JssStyles => ({
     ...sectionTitleStyle(theme),
     display: "inline",
     marginRight: "auto",
+    [theme.breakpoints.down("sm")]: {
+      marginTop: theme.spacing.unit*3,
+    },
```

Comment:
Since `SectionTitle` doesn't expose the top margin property directly (only through a `noTopMargin` prop), I've used this workaround to add the margin back on smaller screens.